### PR TITLE
Fix an empty prerequisites section in update-daemon-set.md 

### PR DESCRIPTION
**Pull request by able8**
The "Before you begin" section is empty, adding the `{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}` to fix it.

---

### CODE REVIEW

**Comment by able8** [comment]
The Preview https://deploy-preview-29080--kubernetes-io-main-staging.netlify.app/docs/tasks/manage-daemon/update-daemon-set/

File: content/en/docs/tasks/manage-daemon/update-daemon-set.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ This page shows how to perform a rolling update on a DaemonSet.
 
 ## {{% heading "prerequisites" %}}
 
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```
Comment (sftim) [inline content/en/docs/tasks/manage-daemon/update-daemon-set.md:14]:
```suggestion
{{< include "task-tutorial-prereqs.md" >}}
```
You don't need to check your version of Kubernetes and also the front matter doesn't specify a minimum cluster version.

**Review by sftim — COMMENTED** [review]
Thanks, but the version check shortcode isn't needed. Otherwise: a good fix.
/lgtm
(but it's even better to omit the version check)

File: content/en/docs/tasks/manage-daemon/update-daemon-set.md (hunk 1 above)
Comment (able8) [reply content/en/docs/tasks/manage-daemon/update-daemon-set.md]: Thanks. Updated.

**Comment by able8** [comment]
Thanks. Updated to omit the version check.

**Pull request merged**